File: src/display/banner.js
import chalk from 'chalk';

export function displayBanner() {
  console.log(chalk.cyan(`
╔══════════════════════════════════════╗
║           ${chalk.bold('BUGSCANNER JS')}             ║
╚══════════════════════════════════════╝
  ${chalk.gray('Server and Protocol Scanner')}
  ${chalk.gray('Inspired by aztecrabbit/bugscanner')}
`));
}

File: src/utils/banner.py
from rich.console import Console
from rich.panel import Panel

def display_banner() -> None:
    console = Console()
    banner = """
[cyan]╔══════════════════════════════════════╗
║           [bold]BUGSCANNER PY[/bold]             ║
╚══════════════════════════════════════╝[/cyan]
  [dim]Server and Protocol Scanner
  Enhanced Python version of bugscanner[/dim]
"""
    console.print(banner)